ui/src/kapacitor/components/RuleHeader.js

```diff
@@ -110,31 +110,31 @@ export const RuleHeader = React.createClass({
           onKeyDown={e => this.handleEditName(e, rule)}
           onBlur={() => this.handleEditNameBlur(rule)}
           placeholder="Name your rule"
+          spellCheck={false}
+          autoComplete={false}
         />
-      : <h1
-          className="page-header__title page-header--editable kapacitor-theme"
-          onClick={this.toggleEditName}
-          data-for="rename-kapacitor-tooltip"
-          data-tip="Click to Rename"
-        >
-          {rule.name}
-          <span className="icon pencil" />
-          <ReactTooltip
-            id="rename-kapacitor-tooltip"
-            delayShow={200}
-            effect="solid"
-            html={true}
-            offset={{top: 2}}
-            place="bottom"
-            class="influx-tooltip kapacitor-tooltip place-bottom"
-          />
-        </h1>
+      : <div className="page-header__left">
```

Comment:
I had set the input to `flex: 1 0 0;` but when it is enclosed within `div.page-header__left` it doesn't actually fill the space of the entire header. Removed the wrapper from the input to achieve the desired appearance